consumers/src/main/java/uk/gov/moj/sdt/cmc/consumers/request/UpdateType.java

```diff
@@ -1,4 +1,8 @@
 package uk.gov.moj.sdt.cmc.consumers.request;
 
-public interface CMCRequest {
+public enum UpdateType {
```

Comment:
Just a note that the CMCRequest class has been deleted in #113, so there may be merge issues with this rename.

Reply:
should be fine